### PR TITLE
ss-321: user settings page redesign

### PR DESCRIPTION
**Pull request by ValeriiArt**
image

### CODE REVIEW

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 1]
```diff
@@ -24,10 +23,10 @@ const currency = [
     { value: 'UAH', name: 'UAH' },
 ];
 
-const language = [
-    { value: 'English', name: 'English' },
-    { value: 'Ukrainian', name: 'Ukrainian' },
-];
+// const language = [
```
Comment (Evcat6) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:26]: Does we need this?

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 2]
```diff
@@ -45,13 +44,12 @@ const SettingsForm: React.FC = () => {
...
     const isChange = (): boolean => {
-        const { firstName, lastName, email, language, currency, sex } =
-            mockData;
+        const { firstName, lastName, email, currency, sex } = mockData;
         return (
             newName !== firstName ||
             newSurname !== lastName ||
             newEmail !== email ||
-            language !== selectedSingleLanguage.name ||
+            // language !== selectedSingleLanguage.name ||
```
Comment (Evcat6) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:52]: The same

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 3]
```diff
@@ -69,17 +67,17 @@ const SettingsForm: React.FC = () => {
         [],
     );
 
-    const [selectedSingleLanguage, setSelectedSingleLanguage] =
-        useState<DataType>(language[0]);
+    // const [selectedSingleLanguage, setSelectedSingleLanguage] =
```
Comment (Evcat6) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:70]: Same

File: frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx [hunk 4]
```diff
@@ -137,21 +135,21 @@ const SettingsForm: React.FC = () => {
...
+
             <Dropdown
                 data={currency}
                 handleChange={handleDropdownChangeCurrency}
                 selectedOption={selectedSingleCurrency}
                 label="Account currency"
                 labelClassName={styles.dropdownLabel}
             />
-            <Dropdown
+            {/* <Dropdown
```
Comment (Evcat6) [inline frontend/src/bundles/account-settings/components/settings/user-profile/settings-form.tsx:146]: Same

**Review by Evcat6 — CHANGES_REQUESTED** [review]

**Review by Evcat6 — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**